src/PKSim.Core/Snapshots/Mappers/PopulationAnalysisFieldMapper.cs

```diff
@@ -100,14 +102,15 @@ private void mapQuantityFieldToSnapshot(PopulationAnalysisField snapshot, IQuant
       {
          mapNumericFieldToSnapshot(snapshot, field);
          snapshot.QuantityPath = field.QuantityPath;
-         snapshot.QuantityType = field.QuantityType;
+         snapshot.QuantityType = field.QuantityType.ToString();
       }
 
       private void mapQuantityFieldToModel(PopulationAnalysisField snapshot, IQuantityField field)
       {
          mapNumericFieldToModel(snapshot, field);
          field.QuantityPath = snapshot.QuantityPath;
-         field.QuantityType = ModelValueFor(snapshot.QuantityType, field.QuantityType);
+         if (snapshot.QuantityType != null)
```

Comment:
Using string here because of problem with schema parsing. This is commented in the actual PopulationAnalysisField  snapshot class